### PR TITLE
feat: update colors

### PR DESCRIPTION
**Pull request by mdole**
I know we're mid-transition to tokens (#818), but not having these new colors was blocking other work for me so I figured I could update in both places and we'd be good for now.

Will also put up a PR in Eigen for the same reason.

Slack context: https://artsy.slack.com/archives/C9XJKPY9W/p1605904957010300
Notion colors page: https://www.notion.so/artsy/Color-a0c24896daf8433d9409aee2146ac267

### CODE REVIEW

File: packages/palette-tokens/src/index.ts [hunk 1]
```diff
@@ -84,25 +84,35 @@ const tokens = {
     /** 10 black (grey), borders, divider lines, and grey button only */
     black10: "#E5E5E5",
     /** 5 black (light grey), backgrounds only */
-    black5: "#F8F8F8",
-    /** 100 Blue, highlight */
+    black5: "#F2F2F2",
```
Comment (mdole) [inline packages/palette-tokens/src/index.ts:87]: @deliciousnachos just to confirm, is `#F2F2F2` the correct value for `black5`? That's what's in Notion, but I wasn't sure if Palette was outdated or Notion was outdated :)

**Comment by damassi** [comment]
Looks good 👍

File: packages/palette-tokens/src/index.ts (hunk 1 above)
Comment (deliciousnachos) [reply packages/palette-tokens/src/index.ts:87]: @mdole Oh interesting! Didn't mean to make any updates to the grays, let's keep what's already in palette #F8F8F8

**Review by damassi — APPROVED** [review]
After color comment has been addressed, merge when ready 👍

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette-tokens@1.1.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette-tokens%401.1.0), [`@artsy/palette@13.23.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4013.23.0) :rocket:

**Pull request merged**